### DISCUSSION TITLE
Initial Website Launch Date

### DISCUSSION THREAD

**Discussion by colinkiama** [nahuelwexd/vala-website#52] [Announcements]
While all the feedback and suggestions are appreciated, we do need to release something for the general public eventually. So I’m setting a launch date: [July 15th 2022](https://gitlab.gnome.org/GNOME/vala/-/commit/68986811db7b23c1c3b652cbee34fd45c62c2c6e).

As we get closer to the date, suggested additions won’t be implemented until after the launch (There will be a label added to these issues that indicate this), bug fixes will be prioritised and this repo will be moved under the [vala-lang organisation](https://github.com/vala-lang).

**Comment by lw64** [comment]
The Vala Talk at GUADEC will be at the July 22nd 2022, 10.40.
https://events.gnome.org/event/77/contributions/298/

**Reply by pervoj** [reply]
@Prince781 It would be good to include the website in the presentation as one of the proofs that Vala is still alive.

**Reply by Prince781** [reply]
Yes I will definitely include this.

**Reply by colinkiama** [reply]
Would you mind if there was post on the site mentioning this event?

**Reply by lw64** [reply]
Yes, I am planning to write a blog post about the new stuff.
